lib/broccoli/angular-broccoli-less.js

```diff
@@ -0,0 +1,69 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+try {
+  let less;
+
+  if (process.platform === 'win32') {
+    require.resolve(`${process.env.PWD}/node_modules/less`);
```

Comment:
Since you are not using the output of `require.resolve()`, then why do you need it ?
`require(...)` should be enough, right ?